### ISSUE TITLE
False positive for exit after wp_redirect()

### ISSUE THREAD

**Issue by GaryJones** [Automattic/VIP-Coding-Standards#410]
## Bug Description

Check for `exit` after `wp_redirect()` doesn't seem to allow for potential third arg in `wp_redirect()`.
## Minimal Code Snippet

```php
wp_redirect( $url, 301, 'X-Foo-Redirect-Post` );
exit;
```

## Error Code

`WordPressVIPMinimum.VIP.ExitAfterRedirect.NoExit`

## Environment

| Question                | Answer
| ------------------------| -------
| PHP version             | N/A
| PHP_CodeSniffer version | Latest
| VIPCS version           | 0.4.0

## Tested Against `master` branch?

- [ ] I have verified the issue still exists in the `master` branch of VIPCS.

**Comment by GaryJones** [comment]
The issue was that a backtick rather than a quote was used. Not a bug in VIPCS.
